frontend/config/chains.ts

```diff
@@ -53,7 +53,7 @@ export const ETHEREUM_CHAIN_CONFIG: ChainConfig = {
 export const CHAIN_CONFIG: {
   [evmChainId: number]: ChainConfig;
 } = {
-  // [EvmChainId.Base]: BASE_CHAIN_CONFIG,
+  [EvmChainId.Base]: BASE_CHAIN_CONFIG,
```

Comment:
enabling this making the balance provider fetch balances on all the chains from this config, however it might not make sense for one-chain agent types, e.g. we don't need to fetch gnosis balances being on memeooor; might be only useful for logging... but unsure.